### PR TITLE
Create CONTRIBUTING.rst

### PR DESCRIPTION
**Pull request by wildsm**
This adds a basic Contributing

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Contributing to the BAND Framework
```
Comment (thchang) [inline CONTRIBUTING.rst:1]: Change to ParMOO

**Comment by thchang** [comment]
Also, should we add a line in the README referencing the CONTRIBUTING.rst document?  Maybe under Resources section?

**Comment by wildsm** [comment]
Yes good idea.
While at t, should probably drop 
https://github.com/parmoo/parmoo/workflows/parmoo/badge.svg?/branch=develop
for now (or at least change to main branch?)

**Comment by thchang** [comment]
Yes, change to main... I'll figure out how to generate the badge

**Comment by thchang** [comment]
Looks like if you change the URL to 

https://github.com/parmoo/parmoo/workflows/parmoo-ci.yml/badge.svg?/branch=main

and then we activate CI for main as well, it should work

https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

**Comment by thchang** [comment]
Can you add main on lines 4 and 6 of .github/workflows/parmoo-ci.yml in addition to above changes?

**Comment by thchang** [comment]
Made some minor corrections to your edits to the workflow file, contributing, and readme
 - keep CI on develop, in addition to main
 - reformat CONTRIBUTING.rst to use static refs at the bottom so that it renders readable in a terminal window
 - add refs to CONTRIBUTING.rst and LICENSE in README.rst

**Review by thchang — APPROVED** [review]

**Pull request merged**